### ISSUE TITLE
Surface trip-based alerts

### ISSUE THREAD

**Issue by devinbraun** [OneBusAway/onebusaway-android#938]
**Summary:** 

OBA Android doesn't show an indication that a trip-level alert exists

**Steps to reproduce:** 

View the arrivals including a trip with an active alert.

**Expected behavior:** 

Show an indication that an alert is active for the trip

**Observed behavior:** 

Nothing, no alert was indicated.

OBA Web does show an alert
![image](https://user-images.githubusercontent.com/5028299/49240929-43feb500-f3bb-11e8-84f9-c451a0b55d6a.png)

iPhone indicates an alert once the specific trip is opened, but nothing on the arrivals view for a stop
![oba_ipad_screenshot](https://user-images.githubusercontent.com/5028299/49241193-f6367c80-f3bb-11e8-8259-bae5a9e20f10.jpg)

**Comment by barbeau** [comment]
Thanks @devinbraun!  Could you provide a specific stop/route/trip where a trip_id is active?  I need to get a snapshot from the OBA APIs of what the response looks like for these stops/trips.

Off the top of my head, I believe you're correct that alerts don't show up in the "Show trip details" view.

Are you sure that the trip-level alert doesn't appear at the top of the arrivals list?  I have a separate issue open for flagging route-specific alerts directly on the arrival item in the list (in addition to showing them at the top of the list with stop-specific alerts) - https://github.com/OneBusAway/onebusaway-android/issues/702.

**Comment by devinbraun** [comment]
Yes, the trip-level alert does show at the top of the screen.  But if it applies only to one trip, I am requesting that it is marked somehow on the actual trip.  Here's the API output for a trip-level alert.
[trip-level alert OBA API output.txt](https://github.com/OneBusAway/onebusaway-android/files/2630464/trip-level.alert.OBA.API.output.txt)

**Comment by barbeau** [comment]
>But if it applies only to one trip, I am requesting that it is marked somehow on the actual trip.

Sounds good! Yes, that's #702. 

I'll leave this issue open for showing alerts in the trip details screen.

**Comment by bmander** [comment]
Implemented — trip/route service alerts show as a tappable indicator on affected arrival rows (`ArrivalRows.kt` `ArrivalAlertIndicator`, driven by `ArrivalInfo.situationIds`), covered by `ArrivalAlertIndicatorTest`. Closing.
